course/content.py

```diff
@@ -527,6 +527,14 @@ class FlowRulesDesc(Struct):
 
 # {{{ mypy: flow
 
+class TabDesc(Struct):
```

Comment:
Needs to be added to the documentation.

Reply:
In my experience, just stick to the truth. Explain the whole thing with ProctorU.

Reply:
Sure, added the explanation